src/components/Popover/Popover.tsx

```diff
@@ -28,46 +17,21 @@ export const Popover = ({
     flip = false,
     enablePortal = false,
```

Comment:
Should be true by default

Reply:
I set it to false by default as the use of React.Portal creates an accessibility issue (violation in Level A). So, Portal should be enabled only when the content really needs to pop out, if it's within a container with overflow hidden or above a canvas for example. Decision consciously made by the dev.

Reply:
This is not true, many accessible libraries are doing it so layering is easier (and zIndex issues are gone).

Reply:
And yet we got issues reported linked to the use of Portal. I can't find in any docs a cause of it, maybe the focus sequence was not right... topic I will tackle here properly with time. I can set default to true but will also add a comment in the docs to disable it if any a11y issue pops out. 👍🏼